packages/core/src/RPA/core/logger.py

```diff
@@ -11,6 +12,14 @@
 from RPA.core.helpers import required_param
 
 
+logging.captureWarnings(True)
+
+
+def deprecation(message):
```

Comment:
@osrjv Is this the right place to introduce such utility? (that I foresee to frequently use)

Reply:
Seems like a good enough place